Sources/SwifterSwift/MapKit/MKPolylineExtensions.swift

```diff
@@ -21,6 +21,16 @@ public extension MKPolyline {
         self.init(coordinates: &refCoordinates, count: refCoordinates.count)
     }
 
+    /// SwifterSwift: Create a new MKPolyline from a provided Array of locations.
+    ///
+    /// - Parameter locations: Array of CLLocation(s).
+    convenience init(locations: [CLLocation]) {
+        var refCoordinates = locations.map {
```

Comment:
You can shorten this to 1 line
```suggestion
        var refCoordinates = locations.map { $0.coordinate }
```

Reply:
@trevorphillips why did you mark this as resolved? In case it's not clear, then function should look like this:
```Swift
convenience init(locations: [CLLocation]) {
    var coordinates = locations.map { $0.coordinate }
    self.init(coordinates: &coordinates, count: coordinates.count)
}
```